conf/config-default.yaml

```diff
@@ -196,6 +196,8 @@ nginx_config:                     # config for render the template to generate n
                                    # After the maximum number of requests is made, the connection is closed.
       keepalive_timeout: 60s       # Sets a timeout during which an idle keepalive connection to an upstream server will stay open.
 
+    charset: utf-8                 # Adds the specified charset to the "Content-Type" response header field.
```

Comment:
Personally, it would be more clear if we remove the blank line above.
And it would be better to link the `charset` directive in the comment.

Reply:
Updated.